<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkXnatConnectionPreferencePageControls</class>
 <widget class="QWidget" name="QmitkXnatConnectionPreferencePageControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>419</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>653</width>
    <height>116</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QGroupBox" name="groupBoxProxySettings">
     <property name="title">
      <string>Network Proxy Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="3">
       <widget class="QLineEdit" name="inProxyPort">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Standard-Port: 80</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="proxyPasswordLabel">
        <property name="text">
         <string>Proxy Password</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="proxyUsernameLabel">
        <property name="text">
         <string>Proxy Username</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="inProxyAddress">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Examples:&lt;/p&gt;&lt;p&gt;https://central.xnat.org&lt;br/&gt;http://localhost/xnat&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="placeholderText">
         <string>http(s)://</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="proxyPortLabel">
        <property name="text">
         <string>Proxy Port</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="inProxyPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>testpassword</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="inProxyUsername">
        <property name="placeholderText">
         <string>testuser</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="proxyAddressLabel">
        <property name="text">
         <string>Proxy Server Address</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxXnatSettings">
     <property name="title">
      <string>XNAT Connection Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QLabel" name="xnatPortLabel">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="xnatUsernameLabel">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="xnatPasswortLabel">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="xnatHostAddressLabel">
        <property name="text">
         <string>Server Address</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="inXnatPort">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Standard-Port: 80</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="inXnatUsername">
        <property name="placeholderText">
         <string>testuser</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QToolButton" name="btnDownloadPath">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="inXnatPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>testpassword</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="inXnatDownloadPath"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="xnatDownloadPathLabel">
        <property name="text">
         <string>Download Path</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="inXnatHostAddress">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Examples:&lt;/p&gt;&lt;p&gt;https://central.xnat.org&lt;br/&gt;http://localhost/xnat&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="placeholderText">
         <string>http(s)://</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="cbUseNetworkProxy">
     <property name="text">
      <string>Use Network Proxy</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QGridLayout" name="groupConnectionTest">
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="buttonSpacerLayout">
       <item>
        <widget class="QPushButton" name="xnatTestConnectionButton">
         <property name="text">
          <string>Test Connection</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="xnatTestConnectionLabel">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cbUseSilentMode">
       <property name="text">
        <string>Silent Mode</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>(Suppresses most message popups)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>inXnatHostAddress</tabstop>
  <tabstop>inXnatPort</tabstop>
  <tabstop>inXnatUsername</tabstop>
  <tabstop>inXnatPassword</tabstop>
  <tabstop>inXnatDownloadPath</tabstop>
  <tabstop>btnDownloadPath</tabstop>
  <tabstop>cbUseNetworkProxy</tabstop>
  <tabstop>inProxyAddress</tabstop>
  <tabstop>inProxyPort</tabstop>
  <tabstop>inProxyUsername</tabstop>
  <tabstop>inProxyPassword</tabstop>
  <tabstop>xnatTestConnectionButton</tabstop>
  <tabstop>cbUseSilentMode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
